apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -198,17 +190,26 @@ export class EmailOutputRendererUsecase extends BaseTranslationRendererUsecase {
         : null;
     }
 
+    const stepBodyHtml = await this.processBodyContent({
+      body,
+      payload,
+      dbWorkflow,
+      locale,
+      noHtmlWrappingTags: !!layoutControlsEntity,
```

Comment:
do not generate the wrapping html tags (html, head, body) only when we have the layout